src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -22,6 +22,8 @@
     <NativeAotSupported Condition="'$(CrossBuild)' == 'true' and '$(TargetOS)' != '$(HostOS)'">false</NativeAotSupported>
     <!-- Can't use NativeAOT in source build yet https://github.com/dotnet/runtime/issues/66859 -->
     <NativeAotSupported Condition="'$(DotNetBuildFromSource)' == 'true'">false</NativeAotSupported>
+    <ObjCopyName Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(CrossBuild)' == 'true' and '$(RuntimeIdentifier)' == 'linux-musl-arm64'">llvm-objcopy-15</ObjCopyName>
+    <ObjCopyName Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(CrossBuild)' == 'true' and '$(RuntimeIdentifier)' == 'linux-arm64'">aarch64-linux-gnu-objcopy</ObjCopyName>
```

Comment:
This can also use llvm-objcopy if/when https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/815 is ingested in runtime CI. It has same libc (v2.2x) in rootfs with better llvm-15 toolchain on host.

The problem with llvm-objcopy-9 was that it failed to reduce size of binary, for which we added a complicated introspection https://github.com/dotnet/runtime/blob/b849f828a550e35013b1151b6a764ebf0582bc96/eng/native/configuretools.cmake#L67 (and our custom function `locate_toolchain_exec`)

This bug was fixed in llvm-objcopy-10. I don't think replicating all that "locate objcopy for a given toolchain and fallback to alternative toolchain" code in shell script is worth the effort.

Reply:
> Do we need to worry this will break once @janvorli or someone else decides we need to roll forward to a newer llvm?

Existing patterns, no need to worry. When we update the container image, we bump these versions otherwise the CI fails. e.g. https://github.com/dotnet/runtime/commit/3eacbaf0f85c6a6dc967cb490480dd256b64e5b0 (see change made in crossgen2.csproj). In previous PR with @jkotas, we decided not to complicate NativeAOT build integration by incorporating system introspection to find the tools. So this is in line with "keeping it simple" version.

Reply:
Sounds good, thanks for explaining!